### PR TITLE
8 create hero section of the homepage

### LINKED ISSUE #8: Create : Hero Section of the Homepage
## Summary

Create the Hero Section of the Homepage. What's a hero section? Google it 🤣. [Here](https://www.optimizely.com/optimization-glossary/hero-image/) is an example of what a hero section is. Hero section generally has a CTA button that takes user to the primary functionality page of a project. Do add them.

Design and responsiveness must be readied with Tailwind, snapshot/unit tests should also be written.

### Screenshots

N/A

### Basic example

N/A

### Motivation

Welcoming users with a professional looking hero section and showing them what our website has to offer.

### PR DESCRIPTION
**Pull request by sozinadr**

### CODE REVIEW

File: src/components/HeroSection/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+function Hero() {
+  return (
+    <div className='backdrop-blur-md bg-white/30 h-[100vh] cont text-center text-background'>
```
Comment (ayubabdullah) [inline src/components/HeroSection/Hero.jsx:4]: adding h-screen instead of h-[100vh] doesn't work?

File: src/index.css [hunk 2]
```diff
@@ -1,33 +1,17 @@
...
-
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```
Comment (ayubabdullah) [inline src/index.css:6]: I think we decided on the font family

File: src/index.css [hunk 3]
```diff
@@ -1,33 +1,17 @@
...
-video {
-width:100%;
-height:100%;
-object-fit: contain;
-}*/
-
-
-
+.cont {
+    background: url('https://images.unsplash.com/photo-1561715276-a2d087060f1d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80') no-repeat center/cover;
```
Comment (ayubabdullah) [inline src/index.css:16]: I think adding the background image inside the tailwind config is better

**Review by ayubabdullah — COMMENTED** [review]
Well done guys good work

I just add some comments and suggestions, reviewing them will be great

File: src/components/HeroSection/Hero.jsx (hunk 1 above)
Comment (OmarBassatni97) [reply src/components/HeroSection/Hero.jsx:4]: It works but its the same

**Review by OmarBassatni97 — APPROVED** [review]
LGTM

**Comment by sarahaboudalal** [comment]
Looks very nice! I only suggest decreasing the width on small screens. Other than that, it's amazing, well done👏🏼

**Pull request merged**